content/docs/sdk/flutter/at_client_manager.md

```diff
@@ -21,9 +21,9 @@ Based on your need you can either  install at_client_mobile or at_client and imp
 AtClientManager.getInstance().setCurrentAtSign('@alice', 'wavi', <preference>);
 ```
 
- The setCurrentAtSign method accepts the following arguments: currentAtSign, namespace and the preferences. The currentAtsign argument accepts an atSign , namespace accepts the namespace of the app and the preference accepts AtClientPreference value which we are going to discuss next.
+ The setCurrentAtSign method accepts the following arguments: currentAtSign, namespace and the preferences. The currentAtsign argument accepts an atsign , namespace accepts the namespace of the app and the preference accepts AtClientPreference value which we are going to discuss next.
```

Comment:
atSign should remain with capital S when referring to the product or string identifier for it.